### DISCUSSION TITLE
how to terminate ghosts on linux?

### DISCUSSION THREAD

**Discussion by kchen1023** [cmu-sei/GHOSTS#371] [Q&A] [answered]
I know its kill-ghosts.bat for windows, but I can't find a similar file for Linux so I've been using ctrl-C and it seems to exit the program. Does this way work properly (will I face any issues using ctrl-c to kill ghosts)?
![Screenshot 2024-06-17 161533](https://github.com/cmu-sei/GHOSTS/assets/131231802/9c6c9a37-d620-46e5-91f9-d171548b7ef4)

**Comment by sei-dupdyke** [comment, marked as answer]
Ctrl-C is fine, won't hurt anything.
